CMakeLists.txt

```diff
@@ -559,17 +559,15 @@ endif()
 # -------------------------------------------------------------
 # setting the RPATH
 # -------------------------------------------------------------
-# use, i.e. don't skip the full RPATH for the build tree
-set(CMAKE_SKIP_BUILD_RPATH FALSE)
-
-# when building, don't use the install RPATH already (but later on when installing)
-set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
```

Comment:
Is not having this variable set going to cause any potential issues on LINUX or mac for that matter if you are just installing normally.

Reply:
It seems to work -- the options just affect the files in the build tree (so the binaries used for testing with ctest). The default values for both options is false, so the it should behave the same as before.